luminoth/models/fasterrcnn/base_config.yml

```diff
@@ -175,13 +175,17 @@ model:
       pre_nms_top_n: 12000
       # Total proposals to use after NMS (sorted by score)
       post_nms_top_n: 2000
+      # Option to apply NMS
+      apply_nms: True
```

Comment:
Maybe all the nms options should be grouped with common "nms" prefix?
I think it could make it a bit more clear.

Same thing with other types of options.

Reply:
Config is loosely mapped to modules. For example, if NMS was a submodule then it would definitely make sense to group all the 'nms options', otherwise it goes against the current structure.

Reply:
This is something to re-evaluate in the future, and would help prevent clashes maybe :)